lib/ipaddr.rb

```diff
@@ -98,6 +98,61 @@ class AddressFamilyError < Error; end
   # Raised when the address is an invalid length.
   class InvalidPrefixError < InvalidAddressError; end
 
+  # The netmask part of the IP address
+  class Netmask
```

Comment:
I think it is an excellent idea to create a class for returning values.  One fundamental concept of IPAddr was to allow manipulating IP addresses without taking care of what family they are in, so I much appreciate this effort and am willing to integrate it.